<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alarm</class>
 <widget class="QWidget" name="alarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>432</height>
   </size>
  </property>
  <widget class="QLabel" name="curtime">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00-00-00 00:00:00:000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加备注</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>是否重复</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l1">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>320</y>
     <width>255</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>281</y>
     <width>132</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>183</y>
     <width>132</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>85</y>
     <width>132</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>232</y>
     <width>132</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>134</y>
     <width>132</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>134</y>
     <width>255</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line4">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>232</y>
     <width>255</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>183</y>
     <width>255</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>85</y>
     <width>255</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line5">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>281</y>
     <width>255</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="set3">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>183</y>
     <width>120</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仅一次</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set5">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>281</y>
     <width>120</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仅一次</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set4">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>232</y>
     <width>120</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仅一次</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set2">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>134</y>
     <width>120</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仅一次</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set1">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>85</y>
     <width>120</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仅一次</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check3">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>183</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check5">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>281</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check2">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>134</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check4">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>232</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check1">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>85</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
